x/gamm/simulation/sim_msgs.go

```diff
@@ -216,7 +216,7 @@ func RandomJoinSwapExternAmountIn(k keeper.Keeper, sim *simtypes.SimCtx, ctx sdk
 
 	// cap joining pool to the pool liquidity
 	newTokenIn := osmoutils.MinCoins(sdk.NewCoins(coinIn), sdk.NewCoins(tokenIn))
-
+	newTokensIn := sdk.NewCoins(newTokenIn[0])
```

Comment:
Please extend it so that sometimes we get one token in, and sometimes multiple tokens in